pages/jdl/applications.md

```diff
@@ -260,6 +260,12 @@ Here are the application options supported in the JDL:
     <td></td>
     <td></td>
   </tr>
+  <tr>
```

Comment:
@jhipster/developers didn't we change the option to blueprints? @murdos

Reply:
my understanding is that `blueprint` is still supported, but `blueprints` are not fully supported yet, see https://github.com/jhipster/jhipster-core/issues/415

Reply:
There's another one, `blueprint` is to be deprecated in favor of `blueprints`.

Reply:
so, should I add `blueprints` and specify that it's a list?

Reply:
(instead of `blueprint`)

Reply:
Ideally, both. The singular version is to be deprecated (but still valid as we speak), blueprints is valid.